### DISCUSSION TITLE
Question about preamble

### DISCUSSION THREAD

**Discussion by adentse** [jopohl/urh#857] [Q&A] [answered]
Hi!

To begin with, I can't seem to find a forum for URH, at least not with a quick search. It's a fantastic piece of software. So I'am asking here.

I'm trying to sniff the protocol between the zappi charger and it's ethernet connected hub. It's a 868MHz communication using the RF69 chip. When adding labels, the labels seems locked to absolute bit-positions. But my recorded data is'nt perfect in the beginning. The preamble might start with 010101... or 1010101... or 001010101... Is there a way to make it find part of the preamble and the syncbytes and then adjust position of everything else according to that? Or should I just edit the signal?

Best regards
Mikael

**Comment by adentse** [comment, marked as answer]
I found the answer myself, and since I'am a good Internet-citizen:

This is the solution. On the "Analysis"-tab in the "Decoding"-dropdown, select "..." at the bottom. Then you can add your own decoding. In my case i don't want to do anything with the data, it's already NRZ. But i dragged the "Cut before/after" from Additional Functions into the "Your decoding"-window. Then selected Cut before (sequence) and added (in binary) part of my preamble and the sync sequence as the sequence and voila!

**Reply by EdwinFairchild** [reply]
this is gold! thanks for the answer!!!!
